### ISSUE TITLE
Add example for TOGL

### ISSUE THREAD

**Issue by marcusblake** [aidos-lab/pytorch-topological#18]
Currently, there are no examples present for usage of TOGL. Would be good to provide example of use case.

**Comment by Pseudomanifold** [comment]
Yes, this is planned. The module is currently a work in progress and I would not consider the API to be stable at the moment.
